src/app/api/services/crud/crud.service.ts

```diff
@@ -59,4 +59,11 @@ export class CrudService {
             url: `${baseUrl}/id/${id}`
         }).pluck('entity');
     }
+
+    public deleteDataById(baseUrl: string, id: string): Observable<any> {
```

Comment:
Missing documentation. Recently we discover that public method doesn't need the ```public``` statement.